jscomp/bsb/bsb_default.ml

```diff
@@ -44,12 +44,12 @@
         https://caml.inria.fr/mantis/view.php?id=6352
 
 *)  
+let warnings = "-40+6+7+27+32..39+44+45"
+
 let bsc_flags = 
   [
     "-no-alias-deps";
     "-color"; "always" ;
-    "-w"; "-40+6+7+27+32..39+44+45"
-
```

Comment:
I added a new field to the bsconfig called `warnings` which takes that ^ values by default, and I added that flag to all commands which uses `bsc_flags`. All this to say that the compiler gets the same flags by default, so no behavioral changes. The idea is that you'll probably want the same warnings on native than on JS. (see my last two comments on this PR)